src/kixi/heimdall/handler.clj

```diff
@@ -54,6 +61,22 @@
       {:status 201 :body res}
       {:status 401 :body res})))
 
+(defn escape-html
+  "Change special characters into HTML character entities."
+  [text]
+  (.. #^String (str text)
+      (replace "&" "&amp;")
+      (replace "<" "&lt;")
+      (replace ">" "&gt;")
+      (replace "\"" "&quot;")))
```

Comment:
hiccup has escape-html, which is exactly this function https://github.com/weavejester/hiccup/blob/53057de9799b27d5d0ddfe865fdac6e7a141d896/src/hiccup/util.clj#L73